src/OpenTelemetry/Metrics/ExponentialHistogramBuckets.cs

```diff
@@ -14,17 +14,21 @@
 // limitations under the License.
 // </copyright>
 
+#nullable enable
+
 namespace OpenTelemetry.Metrics;
 
-internal class ExponentialBuckets
+public class ExponentialHistogramBuckets
 {
     private readonly CircularBufferBuckets buckets;
 
-    internal ExponentialBuckets(CircularBufferBuckets buckets)
+    internal ExponentialHistogramBuckets(CircularBufferBuckets buckets)
     {
         this.buckets = buckets;
     }
 
+    public int Offset => this.buckets.Offset;
```

Comment:
Make this a readonly field?

Reply:
Made it readonly. I originally had it this way, but it wasn't recommending one way or another, so I figured readonly was implied and removed it 🤷.

Reply:
I think there might be some perf benefit of using fields over properties. It shouldn't really matter that much though especially considering this is not the hot path 😄